src/pages/incident/IncidentDetails.tsx

```diff
@@ -206,7 +206,7 @@ export function IncidentDetailsPage() {
               updateIncidentHandler={updateIncidentHandler}
               textButton={status === IncidentStatus.Open ? "Close" : "Reopen"}
             />
-            <Changelog />
+            <Changelog incidentId={incident.id} />
```

Comment:
Why not just pass through the entire incident ?

Reply:
I usually prefer to pass the data which the component requires rather than extra data, which makes the contract clear from the caller perspective on what the component needs to work.